### ISSUE TITLE
improved DL pipeline

### ISSUE THREAD

**Issue by nealgravindra** [nealgravindra/wearables#23]
Want to allow for similarly formatted data to be input to DL pipeline more quickly and take advantage of padding, especially to highlight at least five baselines:

1. kNN-DTW
2. RFTimeSeries 
3. CNN-based (e.g., InceptionTime)
4. RNN-based (e.g., from Palotti et al. 2019)
5. ~ConvLSTM (Rahman et al. 2019)
6. LightGBM
6. Ours (InceptionTime + Performer or Enformer, as new ConViT for time-series)

The only parameter should be the extent of time going into the data from the start of the day, e.g., 1wk of actigraphy data.

This leaves to do:
- [x] cleaned up data access pipeline on nalab server
- [x] read raw with pyactigraphy and add simple sleep v. wake annotations (follow tutorials)
- [x] clean up md for visualization showing summary stats, viz of time-series, and cohort stratified by ptb and sleep qual
- [ ] add more md vars from the larger raw table
- [ ] hyperparam search training w/fixed epoch, simple storage, and save the eval for later after getting model output
- [x] newer eval scheme in eval.py for clf and IG
- [ ] data augmentation scheme (or at least noise injection) for training
